README.md

```diff
@@ -1,6 +1,9 @@
-[![Test Status][test-img]][test]
+# GitHub Action for Vulnerability Scanning
```

Comment:
I thought I saw elsewhere we were going for sentence case? Or possibly since this is the title, we have an exception for title case?

Reply:
@kzantow I don't have a strong opinion! Since this is the title of the README itself, I chose this case, but I'm happy to change it! Let me know. (What was there before was a capital "S" in "scanning", but a lowercase "v" in "vulnerability")